src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -307,8 +308,11 @@ private async Task<PullRequest> SubmitPRAsync(string packageId, string version,
             Reference newBranch = null;
             try
             {
-                // Create new branch synced to upstream master
-                await this.github.Git.Reference.Create(repo.Id, new NewReference($"refs/{newBranchNameHeads}", upstreamMasterSha));
+                var retryPolicy = Policy.Handle<ApiException>().WaitAndRetryAsync(3, i => TimeSpan.FromSeconds(2));

```

Comment:
I'd change this slightly to TimeSpan.FromSeconds(i)

That gives us a bit of a linear backoff with fastFirst behavior ([https://github.com/Polly-Contrib/Polly.Contrib.WaitAndRetry/#retry-first-failure-fast](https://github.com/Polly-Contrib/Polly.Contrib.WaitAndRetry/#retry-first-failure-fast))

Reply:
Changed to suggested